dashboards/kubernetes/node.json

```
@@ -0,0 +1,300 @@
+{
+  "metric_keys": [
+    "node_cpu_usage_nano_cores"
+  ],
+  "dashboard": {
+    "title": "Kubernetes Nodes",
+    "description": "",
+    "visuals": [
+      {
+        "title": "Node CPU Usage",
+        "description": "node_cpu_usage_nano_cores",
+        "line_label": "%name% %node%",
```

We shouldn't include the metric name unless there are multiple metrics in a graph, and for those graphs we're better off using tags on one metric. See my other comment.
```suggestion
        "line_label": "%node%",
```